### PR TITLE
[Horovod Unittest] - Script Fix

### PR DESCRIPTION
**Pull request by DEKHTIARJonathan**
@pkanwar23 I fixed the code and tested in the container: `nvidia/cuda:11.0-devel-ubuntu18.04` it works 
@tgaddair I switched the script to use master as you requested ;)

### CODE REVIEW

File: tensorflow/tools/ci_build/horovod/gpu/nightly.sh [hunk 1]
```diff
@@ -45,29 +44,30 @@ wget https://download.open-mpi.org/release/open-mpi/v4.0/openmpi-4.0.4.tar.gz
...
+./configure --enable-orterun-prefix-by-default
 
 # Install open-mpi.
-sudo make all install
+make -j $(nproc) all
+make install
 export LD_LIBRARY_PATH=/usr/local/lib/openmpi
 sudo ldconfig
 
-sudo update-alternatives --install /usr/bin/gcc gcc /dt7/usr/bin/gcc 60 --slave /usr/bin/g++ g++ /dt7/usr/bin/g++
```
Comment (pkanwar23) [inline tensorflow/tools/ci_build/horovod/gpu/nightly.sh:60]: your environment is not the same as ours. we need this to make sure that the right version of g++ is installed.

**Review by pkanwar23 — COMMENTED** [review]
added comments.

**Comment by DEKHTIARJonathan** [comment]
@pkanwar23 I added it back.
This line made the install to fain on my side. If it keeps on fail. I would advise to look into that line

**Comment by gbaned** [comment]
@DEKHTIARJonathan Can you please resolve conflicts? Thanks!

**Pull request closed without merging**